pillar.example

```diff
@@ -17,13 +17,7 @@ postgres:
   # Append the lines under this item to your postgresql.conf file.
   # Pay attention to indent exactly with 4 spaces for all lines.
   postgresconf: |
-    listen_addresses = '*'  # listen on all interfaces
-
-  # Backup extension for postgresql.conf file, defaults to ``.bak``.
-  # Set to False to stop creation of backup on postgresql.conf changes.
-  {%- if 'status.time' in salt.keys() %}
-  postgresconf_backup: ".backup@{{ salt['status.time']('%y-%m-%d_%H:%M:%S') }}"
-  {%- endif %}
+      listen_addresses = '*'  # listen on all interfaces
```

Comment:
And since it is just a multi-line string which is directly used in the `file.blockreplace` `context` parameter in the same way (literal style), there should be no `-` list indicator.